code/modules/reagents/chemistry_reactions/other.dm

```diff
@@ -43,7 +43,7 @@
 
 /datum/chemical_reaction/pttoxin
 	name = "Toxin"
-	results = list(/datum/reagent/toxin/pttoxin = 2)
+	results = list(/datum/reagent/water = 2)
```

Comment:
This doesn't look intended?

Reply:
It's intended. CM put in an anti-powergame toxin for paracetamol and tramadol.

This removes the toxin, but leaves the reaction into something harmless, so you no longer get punished besides losing the painkiller that reacted.

Reply:
changed

Reply:
We could use pain-inducing chems instead of things that kill you or outright water.

Reply:
> We could use pain-inducing chems instead of things that kill you or outright water.

I could code up a quick pain-inducing chem, sure, but I'd rather not, because that effect's covered by losing the painkiller that was in you when you take the other one.

As for neurotox/toxin, a pain-inducing chem would EITHER make the reaction worthless (so these pills prevented me from taking 15 halloss/tick at the cost of 100-200 pain instantly?) OR make it ubiquitous (I don't take halloss damage at the cost of 50 pain and no lasting damage? Sign me up!)

I think toxin's fine as-is, cause it can be outhealed by 1 tricord injector for ~2-3 min, and antitox straight-up removes it while healing the damage within ~30 seconds.